pkg/orderers/orderer.go

```diff
@@ -115,10 +114,10 @@ func NewOrdererModule(
 	segment *Segment,
```

Comment:
Update the comment describing the parameters of the `NewOrdererModule` function, it's very outdated (or create an issue for doing it).

Reply:
done